sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -205,6 +209,15 @@ def __init__( # pylint: disable=too-many-statements
         if self.aad_credentials:
             scope_override = os.environ.get(Constants.AAD_SCOPE_OVERRIDE, "")
             account_scope = base.create_scope_from_url(self.url_connection)
+            # Calling get_token to warmup the cache
+            start_ns = time.time_ns()
+            account_scope = base.create_scope_from_url(self.url_connection)
+            _logger.info("sync get_token call start: %s | account_name: %s", str(start_ns), str(account_scope))
+            self.aad_credentials.get_token(Constants.AAD_DEFAULT_SCOPE)
```

Comment:
get_token_info - and account_scope - assuming that is the one calculated based on Env variables etc.